tests/modules/individuals/resources/test_individual_crud.py

```diff
@@ -369,7 +369,7 @@ def test_individual_mixed_edm_houston_patch(
         researcher_1,
         individual_id,
         [
-            {'op': 'replace', 'path': '/timeOfBirth', 'value': '1445410800000'},
+            {'op': 'replace', 'path': '/timeOfBirth', 'value': '2000-01-02T03:04:05'},
             {'op': 'replace', 'path': '/timeOfDeath', 'value': 'cedric'},
```

Comment:
This is not being caught in Houston. It seems that there needs to be timeOf validation

Reply:
fixed in latest commit.